src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,20 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin
```

Comment:
`fsnative` is not defined relative to the 1mm 256^3 MRI, but is defined relative to the AC of the individual participant.

Reply:
I'm not sure I follow. Isn't `fsnative` just the xyz coordinates in RAS space  based on the `vox2ras` affine transformation?

Reply:
RAS is only the specification of the direction of the axes, the origin also needs to be defined. That origin is at the anterior commissure, or not?

Reply:
Ah yes you're right. I just noticed I had two incongruent sentences. Adjusted now to specify AC.